test/cli/0003.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+name=$(basename $0)
+root="$(dirname $(dirname $(dirname $0)))"
+gomtree=$(readlink -f ${root}/gomtree)
+t=$(mktemp -d ${root}/go-mtree.XXXXXX)
```

Comment:
the reason i put this as making a temp directory in `${root}` in my example was because some `$TMPDIR` are mounted without xattr support. kind of a silly assumption.